library/std/src/lib.rs

```diff
@@ -272,6 +272,9 @@
 //
 // Language features:
 // tidy-alphabetical-start
+
+// because this actually got to beta without being noticed
```

Comment:
The comment is a bit hard to understand without the context... maybe something like this?
```suggestion
// stabilization was reverted after it hit beta
```